test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -1339,6 +1339,32 @@ def forward(self, x):
         x = torch.randn(2, 3, 4).to(torch.int32)
         self.run_test(FloatingPoint(), x)
 
+    @skipIfUnsupportedMinOpsetVersion(11)
+    def test_prim_min(self):
+        @torch.jit.script
+        def list_append(boxes: List[torch.Tensor]):
+            temp = []
+            for i, b in enumerate(boxes):
```

Comment:
perhaps add a comment here pointing out that `enumerate` is creating `prim::min` in the IR graph.